templates/miq-template-ext-db.yaml

```diff
@@ -265,7 +265,7 @@ objects:
   spec:
     ports:
       -
-        name: "postgresql"
+        name: "postgres"
```

Comment:
@fbladilo @carbonin I dropped the `ql` in this PR as well based on https://github.com/bdunne/docker-postgres/commit/fc01fe724886013892aea77502a9e5844e2e9ded
Is that a problem?
Do you want a separate PR before the repo split?
My goal is consistency, don't care whether we add or drop the `ql`, do you have an opinion?